Gemfile.lock

```diff
@@ -76,8 +76,8 @@ GEM
     public_suffix (4.0.6)
     puma (5.5.1)
       nio4r (~> 2.0)
-    racc (1.5.2)
-    rack (2.2.3)
+    racc (1.6.0)
+    rack (1.6.13)
```

Comment:
@mheap Checking in: this is yet another dependency bump that has to downgrade rack. Is this one that we need, or should we close it out?

Reply:
Close it out, `rack` is only used for testing not the production deployment.

I'll do some reading around why we're being downgraded